awx/settings/defaults.py

```diff
@@ -1061,4 +1061,4 @@
 SYSTEM_USERNAME = None
 
 # feature flags
-FLAGS = {}
+FLAGS = {'FEATURE_INDIRECT_NODE_COUNTING': [{'condition': 'boolean', 'value': False}]}
```

Comment:
Naming convention is `FEATURE_*_ENABLED`, so this should be `FEATURE_INDIRECT_NODE_COUNTING_ENABLED`